resources/python.rb

```diff
@@ -0,0 +1,29 @@
+default_action :install
+
+property :pyenv_name, String, name_property: true
+property :python_version, String, default: '2.7.17'
```

Comment:
If we're releasing a major version can we go to Python 3  + the latest graphite

Reply:
@jtschelling Can you please address this?

Reply:
sorry, stuck on getting python 3.8.5 working here and this unfortunately dropped in priority a little for me. troubleshooting it when I can